CHANGELOG.md

```diff
@@ -17,7 +17,11 @@
 * Merge markers lint test - ignore binary files, allow config to ignore specific files [[#1040](https://github.com/nf-core/tools/pull/1040)]
 * New lint test to check if params in `nextflow config` are mentioned in `main.nf` [[#1038](https://github.com/nf-core/tools/issues/1038)]
 * New modules lint test comparing the `functions.nf` file to the template version
+<<<<<<< HEAD
+* Modules installed from alternative sources are put in folders based on the name of the source repository
+=======
 * Added temporary fix to remove warnings about params that get converted from camelCase to camel-case [[#1035](https://github.com/nf-core/tools/issues/1035)]
+>>>>>>> master
```

Comment:
```suggestion
* Modules installed from alternative sources are put in folders based on the name of the source repository
* Added temporary fix to remove warnings about params that get converted from camelCase to camel-case [[#1035](https://github.com/nf-core/tools/issues/1035)]
```

Reply:
I just re-watched a segment of my talk where I did this live for bytesize and realised that I didn't hit Save after fixing the merge conflict.. Came to check here if I had indeed pushed the code with the merge markers still in it and... yup! 🤦🏻 

Sorry! 😆